tests/data/basic3.tsv

```diff
@@ -1,25 +1,27 @@
 #license: "https://creativecommons.org/publicdomain/zero/1.0/"
 #mapping_set_id: https://w3id.org/sssom/mapping/tests/data/basic3.tsv
 #mapping_tool: "https://github.com/cmungall/rdf_matcher"
-#creator_id: "cjm"
+#creator_id:
+#    - "orcid:1234"
 #mapping_date: "2020-05-30"
 #curie_map:
 #  a: "http://example.org/a/"
 #  b: "http://example.org/b/"
 #  c: "http://example.org/c/"
 #  d: "http://example.org/d/"
-#  rdfs: "http://example.org/rdfs/"
-#  owl: "http://example.org/owl/"
+#  rdfs: "http://www.w3.org/2000/01/rdf-schema#"
+#  owl: "http://www.w3.org/2002/07/owl#"
+#  orcid: "https://orcid.org/my-orcid?orcid="

```

Comment:
That is not the correct orcid prefix :P I guess it does not matter, of course, but just FYI.: `https://orcid.org/0000-0002-7356-1779`